Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Biocode/biocode.yml

```
@@ -98,6 +98,26 @@
+  components:
+  - type: FactionWeaponBlocker
+    factions:
+    - Syndicate
+    alertText: Данное оружие биокодировано. Вы не можете его использовать.
+
+- type: entity
+  parent: WeaponSniperDragunov
+  id: WeaponSniperDragunovBiocode
+  suffix: BIOCODE
+  components:
+  - type: FactionWeaponBlocker
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)